frontend/src/Pages/AdminPage/ManageUserPage/DisableUserPage.jsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Divider, Modal, Space } from "antd";
+import { useEffect, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { checkCanDisableUser, disableUser } from "../../../Apis/UserApis";
+
+export function DisableUserPage() {
+  let { id } = useParams();
+  const navigate = useNavigate();
+  const [isModalOpen, setIsModalOpen] = useState(true);
+  const [isAbleToDisable, setIsAbleToDisable] = useState(false);
+
+  useEffect(() => {
+    async function checkValid() {
+      var result = await checkCanDisableUser(id);
+
+      if (result.isSuccess === true) {
+        setIsAbleToDisable(true);
+      } else {
+        setIsAbleToDisable(false);
+      }
+    }
+
+    checkValid();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const handleDisable = async () => {
+    await disableUser({ id });
+
+    setIsModalOpen(false);
+    navigate("/admin/manage-user");
+  };
+
+  const handleOnclick = () => {
+    setIsModalOpen(false);
+    navigate(-1);
+  };
+
+  const onCancel = () => {
+    navigate(-1);
+  };
+
+  return (
+    <>
+      {isAbleToDisable ? (
+        <Modal open={isModalOpen} closable={false} footer={false} width={400}>
```

Comment:
Why did yous set to false here?

Reply:
Because in design there is no close button

Reply:
And I don't want to use the default footer of antd so I set the footer to false